app/import.go

```diff
@@ -146,6 +152,13 @@ func (a *App) bulkImport(c *request.Context, fileReader io.Reader, dryRun bool,
 	var linesChan chan LineImportWorkerData
 	lastLineType := ""
 
+	attachedFiles := make(map[string]*zip.File)
+	if attachmentsReader != nil {
+		for _, fi := range attachmentsReader.File {
```

Comment:
Maybe we could allocate the map only if the zip reader is not nil and leverage the length of `attachmentsReader.File` to specify its capacity.

Reply:
Great idea; done.